akka-http-core/src/main/scala/akka/http/impl/util/LogByteStringTools.scala

```diff
@@ -35,8 +35,8 @@ private[akka] object LogByteStringTools {
 
   def logTLSBidi(name: String, maxBytes: Int = MaxBytesPrinted): BidiFlow[SslTlsOutbound, SslTlsOutbound, SslTlsInbound, SslTlsInbound, NotUsed] =
     BidiFlow.fromFlows(
-      logTlsOutbound(s"$name DOWN", maxBytes),
-      logTlsInbound(s"$name UP  ", maxBytes))
+      logTlsOutbound(s"$name FromNet", maxBytes),
```

Comment:
Here it needs to be swapped  around ;) Thanks to `TlsInbound` / `TlsOutbound` it is quite clear which direction is which.

Reply:
Hah you're right, ctrl+r-ing this change was a bad idea, thanks for checking